contrib/IECoreUSD/src/IECoreUSD/DataAlgo.cpp

```diff
@@ -169,7 +193,7 @@ IECore::DataPtr dataFromSdfAssetPath( const SdfAssetPath &assetPath, const pxr::
 		)
 		{
 			return new StringData(
-				SdfComputeAssetPathRelativeToLayer( spec->GetLayer(), assetPath.GetAssetPath() )
+				SdfComputeAssetPathRelativeToLayer( spec->GetLayer(), p )
```

Comment:
`p` is derived from `GetResolvedPath()`, but we need to continue to pass `GetAssetPath()` here. I think this is most likely responsible for the CI failure.

I also wonder if `SdfComputeAssetPathRelativeToLayer()` might introduces `\` of its own? Maybe it's better to do the substitutions on the result than the argument?

Reply:
Right, that makes sense. Fixed in https://github.com/ImageEngine/cortex/pull/1426/commits/041cfa70f1a50ddc72a01fc4e8628b531940c4d9